spec/path_template_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+RSpec.describe OpenapiFirst::PathTemplate do
+  describe '#match' do
+    it 'returns empty params with exact string match' do
+      expect(described_class.new('/a/b').match('/a/b')).to eq({})
+    end
+
+    it 'returns params with multiple matches' do
+      expect(described_class.new('/{a}/{b}').match('/1/2')).to eq({ 'a' => '1', 'b' => '2' })
+    end
+
+    it 'ignores trailing slashes in paths' do
+      expect(described_class.new('/{a}/{b}').match('/1/2/')).to eq({ 'a' => '1', 'b' => '2' })
+    end
+
+    it 'returns params with kebab-case names' do
+      expect(described_class.new('/{ke-bab}/{under_score}').match('/1/2')).to eq({ 'ke-bab' => '1',
+                                                                                   'under_score' => '2' })
+    end
```

Comment:
I'd add a path with kebab-case just to make sure the names are working as well, wdyt? :)

```suggestion
    it 'returns params with kebab-case names' do
      expect(described_class.new('/kebab-path/{ke-bab}/{under_score}').match('/kebab-path/1/2'))
        .to eq({ 'ke-bab' => '1', 'under_score' => '2' })
    end

```